more_autograding_examples/left_right_exam_seating/config/config.json

```diff
@@ -1,7 +1,7 @@
 {
     "hide_test_details" : true,
 
-    "assignment_message" : "To better accomodate students for the tests and final exam, please let us know if you are left-handed or right-handed.  \nWe will use this information to make the seating assignments.",
```

Comment:
This should probably be "\n We"

Reply:
Doing so will add a space before "We", which may be undesired in some cases.  I think that particular part should be left alone.